org.jacoco.core/src/org/jacoco/core/analysis/Analyzer.java

```diff
@@ -108,6 +108,7 @@ public void visitEnd() {
 	public void analyzeClass(final ClassReader reader) {
 		final ClassVisitor visitor = createAnalyzingVisitor(
 				CRC64.checksum(reader.b), reader.getClassName());
+		// TODO maybe we can SKIP_FRAMES here?
```

Comment:
Yes, but with explicit statement that it will reappear here :wink: - see https://github.com/jacoco/jacoco/pull/501#discussion_r108471442